### PR TITLE
Create event detail page

### LINKED ISSUE #6: Create event detail
API:

- [x] Add an event detail endpoint (public)
- [ ] Make /events/:slug actually query the database

Web: 

- [x] Add an event detail page
- [x] Wire event detail page with the API

### PR DESCRIPTION
**Pull request by alanreid**
Closes #6

### CODE REVIEW

File: web/src/components/event/EventDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Card } from "@blueprintjs/core";
+import { GroupEvent } from "./event-demo";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+import moment from "moment";
+
+import "./EventDetail.scss";
+
+type EventDetailProps = RouteComponentProps<any> & {
```
Comment (iorlandini) [inline web/src/components/event/EventDetail.tsx]: @alanreid I think this looks better:
```
interface EventDetailProps extends RouteComponentProps {
  event: GroupEvent | null;
};

class EventDetail extends Component<EventDetailProps> {
```

Comment (alanreid) [reply web/src/components/event/EventDetail.tsx]: Yeah, although apparently composition is preferred over inheritance in the React world. 
https://reactjs.org/docs/composition-vs-inheritance.html

I don't really find any issues with your code though!

**Comment by alanreid** [comment]
@iorlandini fixed!

File: web/src/components/event/EventDetail.tsx (hunk 1 above)
Comment (iorlandini) [reply web/src/components/event/EventDetail.tsx]: I know, but they are talking about the components and I agree. We are talking about the TypeScript interfaces.

File: web/src/components/event/EventList.tsx [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import React, { Component } from "react";
 import { Card } from "@blueprintjs/core";
+import { Link, withRouter, RouteComponentProps } from "react-router-dom";
 import "./EventList.scss";
 
+type EventListProps = RouteComponentProps<any> & {
```
Comment (iorlandini) [inline web/src/components/event/EventList.tsx]: @alan remember to change this one too!

Comment (alanreid) [reply web/src/components/event/EventList.tsx]: done!

**Pull request merged**